core/server/api/mail.js

```diff
@@ -1,6 +1,6 @@
 // # Mail API
 // API for sending Mail
-var _            = require('lodash'),
+var _            = require('lodash').runInContext(),
```

Comment:
Since we override `_.templateSettings` further down we should use runInContext here: https://github.com/lodash/lodash/issues/705